README.md

```diff
@@ -533,70 +361,78 @@ The above figure shows how to connect the NUCLEO_F429ZI with the v1.0.0 CI test
 
 - To get the SD Card to work with this platform the CI test shield cannot be connected directly to this board, instead follow the instructions above.
 - Any SD-card adapter will work as long as you connect all the relevant pins (MOSI, MISO, SCLK, CS, 3.3V and GND) as illustrated in figure 3.
-- The SDCard is fully inserted into the slot and overhangs the PCB by ~1mm.  
+- The SDCard is fully inserted into the slot and overhangs the PCB by ~1mm.
 
-# SDBlockDevice Example Application 
+# SDBlockDevice Example Application
 
-The following sample code illustrates how to use the sd-driver Block Device API: 
+The following sample code illustrates how to use the sd-driver Block Device API:
 
 
     #include "mbed.h"
     #include "SDBlockDevice.h"
 
-    // Instantiate the SDBlockDevice by specifying the SPI pins connected to the SDCard 
+    // Instantiate the SDBlockDevice by specifying the SPI pins connected to the SDCard
     // socket. The PINS are:
     //     MOSI (Master Out Slave In)
     //     MISO (Master In Slave Out)
     //     SCLK (Serial Clock)
     //     CS (Chip Select)
     SDBlockDevice sd(MBED_CONF_SD_SPI_MOSI, MBED_CONF_SD_SPI_MISO, MBED_CONF_SD_SPI_CLK, MBED_CONF_SD_SPI_CS);
     uint8_t block[512] = "Hello World!\n";
-    
-    int main() 
+
+    int main()
     {
         // call the SDBlockDevice instance initialisation method.
-        sd.init();
-        
-        // Write some the data block to the device 
-        sd.program(block, 0, 512);
-        
-        // read the data block from the device
-        sd.read(block, 0, 512);
-        
-        // print the contents of the block
-        printf("%s", block);
-        
+		if ( 0 != sd.init()) {
+			printf("Init failed \n");
+			return -1;
+		}
+
+		// set the frequency
```

Comment:
This looks like it's using tabs, which is going to show up weird on the README since github renders the page with 8-space tabs and parts of the example use 4-spaces. I would just use 4-spaces for the indention here.